events/2019/11-contributor-summit/README.md

```diff
@@ -1 +1,50 @@
 # 2019 Kubernetes Contributor Summit NA
+
+## *This event is still in the planning stages. Please check back here for updates!*
+
+## What
+
+This event brings together new and current Kubernetes contributors to connect and learn from one another. It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice. The summit spans two days - an optional social event in the evening of Sunday, November 17th along with the main full-day event on Monday, November 18th.
+
+## Registration
+
+- [Sign the CLA](/CLA.md) if you have not done so already.
+- Registration Form Coming Soon
+
+## When and Where
+
+The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon NA](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/), make sure you plan travel accordingly.
+
+- Sunday, November 17th, 2019 - Social event (optional)
+- Monday, November 18th, 2019 - Contributor Summit (New Contributor Workshops and Current Contributor Summit)
+- San Diego, California
+- Event website with more information: LINK TBD
+
+### Transparency
+
+- Recording: Sessions will be video recorded and uploaded to YouTube.
+- Transcription: Transcriptions will be uploaded with the video.
+- Content Sharing: All presentations will be uploaded.
+
+## Team
+
+| Role | Lead | Shadow | Notes |
+|---|---|---|---|
+| Event Lead | Paris | Jeff | |
+| Project Manager | Jamie | himanshu | |
+| Registration | Bob | Nabarun | |
+| Marketing | Jonas | * | |
+| CNCF Events Manager | Deb Giles | N/A | CNCF/LF Employee |
+| Overall Content | Josh | Yang | Additional roles TBD |
+| New Contributor Workshop | Guin | Rael and/or Allison | Dawn Foster to organize SIG Meet and Greet |
+| Accessibility, Inclusiveness, and Diversity | Tasha | Arnaud | |
+
+## Code of Conduct
+
+This event, like all Kubernetes events, has a [Code of Conduct](/code-of-conduct.md). We will have an onsite rep with contact information to be provided here and posted during the event.
+
+## Misc
+
+We want to remove as many barriers as possible for you to attend this event. Please contact community@kubernetes.io to see if we can accommodate a request.
```

Comment:
add #summit-staff as a contact point for folks to reach us or general info and hangout at #contributor-summit

Reply:
Done, although I'm curious if we should link to #summit-staff at all. Realistically we address questions in the #contributor-summit channel already. 

Thoughts?

Reply:
100% ok - squash and go :)